apps/web/src/components/applicant/ApplicantProfile.tsx

```diff
@@ -1,11 +1,42 @@
-import { formatDate } from '@ien/common';
-import { DetailsItem } from '@components';
+import { formatDate, isHired } from '@ien/common';
+import { DetailsItem, OfferAcceptedBanner } from '@components';
 import { useApplicantContext } from './ApplicantContext';
 import { convertCountryCode } from '../../services/convert-country-code';
 import { DetailsHeader } from '../DetailsHeader';
+import hiredIndIcon from '@assets/img/hired_indicator.svg';
+import hiredIndCheckmark from '@assets/img/hired_checkmark_banner.svg';
 
 export const ApplicantProfile = () => {
-  const { applicant } = useApplicantContext();
+  const { applicant, offerWithDiffHa } = useApplicantContext();
+
+  // check for an accepted offer within the same HA as current user
+  const acceptedJobOfferWithSameHa = (): boolean | undefined => {
+    return applicant.jobs?.some(s => s.status_audit?.find(a => isHired(a.status.id)));
+  };
+
+  const showHiredBanner = () => {
+    // show if an offer was accepted within a different HA than logged in user
+    if (offerWithDiffHa) {
```

Comment:
How about moving this conditional controls to OfferAcceptedBanner component? Then we don't need this helper method.